### DISCUSSION TITLE
命令行无法打包，Action打包的Windows包体没有附件可供下载

### DISCUSSION THREAD

**Discussion by jqkj123** [tw93/Pake#454] [Q&A]
![image](https://github.com/tw93/Pake/assets/38273762/77979295-4bc4-4454-a357-48248f22acfb)
如图，命令行中任何网址都没办法完成打包，Rust>1.7，Action打包的windows版没有附件
![image](https://github.com/tw93/Pake/assets/38273762/9602f6f9-0f20-4eb1-8c3f-f4779d2a9c7f)
尝试了很多遍都没有结果，求教！

**Comment by tw93** [comment]
看了一下已经好了，哈哈可以说说原因
<img width="849" alt="image" src="https://github.com/tw93/Pake/assets/8736212/ec384b1b-b041-4e67-a864-bc32ab371644">

**Reply by jqkj123** [reply]
尝试了一下默认图标，在线打包成功了。但是还是不知道为啥图标获取不到，Mac版本也用了同样的icon图标，打包成功，Win版本也给了同样的在线链接，怎么都打包不出来，按照教程用Cache打包也打包不出来。命令行打包也一直报错，十分奇怪
